src/js/AccessibilityUtils.js

```diff
@@ -13,9 +13,10 @@
 // limitations under the License.
 
 goog.require('axs.browserUtils');
+goog.require('axs.color');
+goog.require('axs.color.Color');
```

Comment:
I don't think goog.require actually does anything useful here, to be honest, since you have to specify to the compiler what files to use anyway - it's more like documentation for the developer at this point.

Happy to take out the redundant require if you like; I just like being explicit when pulling in classes (habit from Java/C++ I guess).